webpack/JobWizard/__tests__/integration.test.js

```diff
@@ -1,43 +1,30 @@
 import React from 'react';
 import { Provider } from 'react-redux';
-import configureMockStore from 'redux-mock-store';
 import { mount } from '@theforeman/test';
 import { render, fireEvent, screen, act } from '@testing-library/react';
 import * as api from 'foremanReact/redux/API';
 import { JobWizard } from '../JobWizard';
 import * as selectors from '../JobWizardSelectors';
-import { jobTemplates, jobTemplateResponse as jobTemplate } from './fixtures';
+import {
+  testSetup,
+  mockApi,
+  jobCategories,
+  jobTemplateResponse as jobTemplate,
+} from './fixtures';
 
-jest.spyOn(api, 'get');
-jest.spyOn(selectors, 'selectJobTemplate');
-jest.spyOn(selectors, 'selectJobTemplates');
-jest.spyOn(selectors, 'selectJobCategories');
-jest.spyOn(selectors, 'selectJobCategoriesStatus');
+const store = testSetup(selectors, api);
+
+selectors.selectJobTemplate.mockImplementation(() => {});
 jest.spyOn(selectors, 'selectEffectiveUser');
 jest.spyOn(selectors, 'selectTemplateInputs');
 jest.spyOn(selectors, 'selectAdvancedTemplateInputs');
```

Comment:
should these selectors move to the fixtures file as well?

Reply:
those are leftovers so I removed them now :)